tests/noto_fonts_for_android_test.py

```diff
@@ -158,4 +165,23 @@ def test_font_full_weight_coverage():
 
 
 def test_font_psnames():
-  pass
+  root = etree.parse(str(_noto_4_android_file()))
+  errors = []
+  font_to_xml_psnames = collections.defaultdict(set)
+
+  for font_el in root.xpath("//font[@path]"): 
+    path = _font_path(font_el)
+    psname = _psname(font_el)
+    font_to_xml_psnames[(path, font_el.attrib.get("index", -1))].add(str(psname))
+
+  for (font_path, font_number), xml_psname in font_to_xml_psnames.items():
+    font = _open_font_path(font_path, font_number)
+    postscript_names = get_name_entry_strings(font, _POSTSCRIPT_NAME)
+    if len(postscript_names) != 1:
```

Comment:
It's fine for len to != 1 ... if they are all the same (such as it has both windows and mac name records with the same value). I would wrap get_name_entry_strings in set() and then do the len check.

Reply:
done